crates/core/component/distributions/src/component.rs

```diff
@@ -143,13 +144,39 @@ trait DistributionManager: StateWriteExt {
         // Grab the ambient epoch index.
         let current_epoch = self.get_current_epoch().await?;
 
+        // New issuance for the current epoch.
         let new_issuance = self.compute_new_lqt_issuance(current_epoch).await?;
         tracing::debug!(
             ?new_issuance,
-            "computed new lqt reward issuance for epoch {}",
+            "computed new lqt reward issuance for current epoch {}",
             current_epoch.index
         );
-        Ok(self.set_lqt_reward_issuance_for_epoch(current_epoch.index, new_issuance))
+
+        // Retrieve the previous cumulative LQT issuance total from NV storage.
+        let previous_issuance = self
+            .get_cummulative_lqt_reward_issuance(current_epoch.index - 1)
+            .await
+            .ok_or_else(|| {
+                tonic::Status::not_found(format!(
+                    "failed to retrieve cumulative LQT issuance for epoch {} from non-verifiable storage",
+                    current_epoch.index,
+                ))
+            })?;
+
+        // Compute cumulative LQT issuance up until current epoch.
+        let cumulative_issuance = previous_issuance + new_issuance;
+
+        // Emit an event for LQT pool size increase.
+        self.record_proto(event::event_lqt_pool_size_increase(
```

Comment:
We should emit an event every block, but we don't need to do anything else, the goal for this event is to be a convenient tick for observability, that's all we need imo.